examples/carousel/carousel-2-tablist.html

```diff
@@ -0,0 +1,791 @@
+                              aria-selected="false"
+                              aria-controls="carousel-item-3">
+                        <svg width="34" height="34" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                          <circle class="border" cx="16" cy="15" r="10" />
+                          <circle class="tab-background" cx="16" cy="15" r="8" />
+                          <circle class="tab" cx="16" cy="15" r="6" />
+                        </svg>
+                      </button>
+                      <button id="carousel-tab-4"
+                              role="tab"
+                              tabindex="-1"
+                              aria-label="Slide 4"
+                              aria-selected="false"
+                              aria-controls="carousel-item-4">
+                        <svg width="34" height="34" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                          <circle class="border" cx="16" cy="15" r="10" />
+                          <circle class="tab-background" cx="16" cy="15" r="8" />
+                          <circle class="tab" cx="16" cy="15" r="6" />
+                        </svg>
+                      </button>
+                      <button id="carousel-tab-5"
+                              role="tab"
+                              tabindex="-1"
+                              aria-label="Slide 5"
+                              aria-selected="false"
+                              aria-controls="carousel-item-5">
+                        <svg width="34" height="34" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                          <circle class="border" cx="16" cy="15" r="10" />
+                          <circle class="tab-background" cx="16" cy="15" r="8" />
+                          <circle class="tab" cx="16" cy="15" r="6" />
+                        </svg>
+                      </button>
+                      <button id="carousel-tab-6"
+                              role="tab"
+                              tabindex="-1"
+                              aria-label="Slide 6"
+                              aria-selected="false"
+                              aria-controls="carousel-item-6">
+                        <svg width="34" height="34" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                          <circle class="border" cx="16" cy="15" r="10" />
+                          <circle class="tab-background" cx="16" cy="15" r="8" />
+                          <circle class="tab" cx="16" cy="15" r="6" />
+                        </svg>
+                      </button>
+                    </div>
+                  </div>
+                </div>
+
+
+                <div id="myCarousel-items" class="carousel-items" aria-live="off">
+
+                  <div class="carousel-item active"
+                       id="carousel-item-1"
+                       role="tabpanel"
+                       tabindex="0"
```

Comment:
Can we remove this `tabindex` or remove the `a` inside of the `tabpanel`? It’s super confusing to me that there are two different focusables per tab with no apparent visual/operational difference.

Reply:
@ZoeBijl This is an open issue for discussion in the working group on whether the tabpanel should always received focus.   RIght now the regression test for tab panel require using the tabindex=0 on the tabpanel.   So the main question is predictability on the part of the screen reader users, should they always expect a tab stop on the tabpanel, or not.    So there is currently a disconnect between the authoring guidance and the regression test for this issue.   I think it should just remain until the group decides whether tab stop on tabpanel is required or optional and we get the regression tests and the practices lined up.

Reply:
In that case, can we remove the `a`s inside the panels? They don’t seem to do anything other than add a tabstop that confuses people.